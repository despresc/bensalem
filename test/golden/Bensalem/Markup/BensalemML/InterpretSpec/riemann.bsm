\#section

\heading{Über die Anzahl der Primzahlen unter einer gegebenen Größe}

\##section

\&p
  Meinen Dank für die Auszeichnung, welche mir die Akademie durch die Aufnahme
  unter ihre Correspondenten hat zu Theil werden lassen, glaube ich am besten
  dadurch zu erkennen zu geben, daß ich von der hierdurch erhaltenen Erlaubniß
  baldigst Gebrauch mache durch Mittheilung einer Untersuchung über die Häufigkeit
  der Primzahlen; ein Gegenstand, welcher durch das Interesse, welches
  \personRef{Gauss} und \personRef{Dirichlet} demselben längere Zeit geschenkt haben,
  einer solchen Mittheilung vielleicht nicht ganz unwerth erscheint.

\&p
  Bei dieser Untersuchung diente mir als Ausgangspunkt die von
  \personRef{Euler} gemachte Bemerkung, daß das Product
  \&tdmath
    \`|\prod\frac 1{1 - \frac 1{p^s}} = \sum\frac 1{n^s}\text{,}|`/

\&p
  \physPage{672} wenn für \timath{p} alle Primzahlen, für \timath{n} alle ganzen
  Zahlen gesetzt werden. Die Function der complexen Veränderlichen \timath{s},
  welche durch diese beiden Ausdrücke, so lange sie convergiren, dargestellt wird,
  bezeichne ich durch \timath{\\zeta(s)}. Beide convergiren nur, so lange der
  reelle Theil von \timath{s} größer als \timath{1} ist; es läßt sich indeß leicht
  ein immer gültig bleibender Ausdruck der Function finden. Durch Anwendung der
  Gleichung
  \&tdmath
    \`|∫_0^∞ e^{-nx} x^{s-1} dx = \frac{\Pi(s-1)}{n^s}|`/
  erhält man zunächst
  \&tdmath
    \`|\Pi(s-1)\zeta(s) = ∫_0^∞ \frac{x^{s-1} dx}{e^x - 1}\text{.}|`/
