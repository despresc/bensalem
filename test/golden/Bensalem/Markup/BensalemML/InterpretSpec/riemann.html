<section>

<h1>Über die Anzahl der Primzahlen unter einer gegebenen Größe</h1>

<section>

<p>
  Meinen Dank für die Auszeichnung, welche mir die Akademie durch die Aufnahme
  unter ihre Correspondenten hat zu Theil werden lassen, glaube ich am besten
  dadurch zu erkennen zu geben, daß ich von der hierdurch erhaltenen Erlaubniß
  baldigst Gebrauch mache durch Mittheilung einer Untersuchung über die Häufigkeit
  der Primzahlen; ein Gegenstand, welcher durch das Interesse, welches
  <span class="personRef">Gauss</span> und <span class="personRef">Dirichlet</span> demselben längere Zeit geschenkt haben,
  einer solchen Mittheilung vielleicht nicht ganz unwerth erscheint.</p>

<p>
  Bei dieser Untersuchung diente mir als Ausgangspunkt die von
  <span class="personRef">Euler</span> gemachte Bemerkung, daß das Product
  $$
    \prod\frac 1{1 - \frac 1{p^s}} = \sum\frac 1{n^s}\text{,}$$</p>

<p>
  <span class="physPage">672</span> wenn für \(p\) alle Primzahlen, für \(n\) alle ganzen
  Zahlen gesetzt werden. Die Function der complexen Veränderlichen \(s\),
  welche durch diese beiden Ausdrücke, so lange sie convergiren, dargestellt wird,
  bezeichne ich durch \(\zeta(s)\). Beide convergiren nur, so lange der
  reelle Theil von \(s\) größer als \(1\) ist; es läßt sich indeß leicht
  ein immer gültig bleibender Ausdruck der Function finden. Durch Anwendung der
  Gleichung
  $$
    ∫_0^∞ e^{-nx} x^{s-1} dx = \frac{\Pi(s-1)}{n^s}$$
  erhält man zunächst
  $$
    \Pi(s-1)\zeta(s) = ∫_0^∞ \frac{x^{s-1} dx}{e^x - 1}\text{.}$$</p></section></section>
